syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTNymIDSource";
option optimize_for = LITE_RUNTIME;

import public "Enums.proto";
import public "AsymmetricKey.proto";
import public "PaymentCode.proto";

message NymIDSource {
    optional uint32 version = 1;
    optional SourceType type = 2;
    oneof source {
        AsymmetricKey key = 3;
        PaymentCode paymentcode = 4;
    }
}
